src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+This stands for all HTTP params types (including variables in the body) except query params which are defined in the following variable.
+* queryParams: A JSON collection of name/value pairs which holds the name of the query param to replace and its value.
+| A RESTResponseV2 object. Go to ServiceNow's link:https://developer.servicenow.com/dev.do#!/reference/api/quebec/server/sn_ws-namespace/c_RESTResponseV2API[RESTResponseV2 API documentation]
+to get a list of its functions.
+|Cell in column 4, row 3
+|===
+
+*JobRepository*: It is the service class which holds functions to perform CRUD operations on the application's table labeled "Submitted job".
+A table is a collection of records in the database. Each record corresponds to a row in a table, and each field on a record corresponds to a column on that table.
+The Submitted Job table can be used to store Job Info data received from ProActive and access it in your pipelines.
+Like other Script Include, it can be instantiated and called anywhere scripting is permitted.
+
+You will find in the next section the structure of the table, which corresponds to JobInfo data fields returned by the Scheduler.
+
+Here is the list of the current available functions
+
+[cols="1,1,1,1"]
+|===
+|Function name | Action | Inputs | Output
+
+| createSubmittedJob
+| Adds a row to the Submitted Job table
+a|
+* sessionId: Session id of the ProActive user
+* jobInfo: JobInfo data object to insert in the table
+| The sys id of the created row
+
+| updateSubmittedJob
+| updates a row to the Submitted Job table
+a|
+* sessionId: Session id of the ProActive user
+* jobInfo: Complete JobInfo data object with updated fields.
+| True if the update has succeeded, else false
+
+| deleteSubmittedJobBySysId
+| Deletes a row which has the specified sys id
+a|
+* sysId: The sys id of the row to be deleted
+| True if the deletion has succeeded, else false
+
+| deleteSubmittedJobByJobId
+| Deletes a row which has the specified job id
+a|
+* jobId: The id of the job to be deleted
+| True if the deletion has succeeded, else false
+
+| _isJobExists
+| Checks if a row exists with the specified job id
+a|
+* jobId: The id of the job to check
+| True if the row exists, else false
+
+| _executeUpdateQuery
+| Utility function to encapsulate the update of a row process
+a|
+* jobId: The id of the job to be updated
+| True if the update has succeeded, else false
+|Cell in column 4, row 3
+|===
+
+===== Tables
+
+The application has two tables that can be used to store and read data:
+
+. *Submitted Job*: Each row contains data of a job submitted to the Scheduler.
+This is the table used by the JobRepository Script Include to perform CRUD operations.
+. *Workflow*: Each row contains a ProActive workflow xml definition file. The function submitJobFromFile of the SchedulerService Script Include
+will search for the specified workflow xml definition in this table.
```

Comment:
xml workflow definition

Reply:
I think it is better specified workflow's XML definition